pnpm-workspace.yaml

```diff
@@ -1,7 +1,7 @@
 packages:
   - app/common
   - app/gui
-  - app/ide-desktop/client
+  - app/client
   - app/ide-desktop/icons
```

Comment:
The `app/ide-desktop/icons` is no more, isn't it?

Also I feel `app/client` is not informative for any contributor straying at our codebase. Maybe `electron-client`?

Reply:
Good catch. `electron-client` sounds good.